dag-sync.sh

```diff
@@ -30,6 +30,10 @@ have_dag=$(git log -p -1 "$new" --pretty=format: --name-only | grep "catalog/dag
 
 # Pull out the subject from the new commit
 subject=$(git log -1 --format='%s')
+# Swap the < & > characters for their HTML entities so they aren't
+# interpreted as delimiters by Slack
+subject=${subject//>/&gt;}
+subject=${subject//</&lt;}
```

Comment:
I would've thought that the `&` or `;` would need another layer of escaping.

Reply:
Me too, but apparently not since it's in `${...}`!